alvr/audio/src/lib.rs

```diff
@@ -66,29 +49,9 @@ fn device_from_custom_config(
     })
 }
 
-// Input and output devices may have the same name.
-fn microphone_pair_from_sink_name(host: &Host, sink_name: &str) -> Result<(Device, Device)> {
-    let sink = host
-        .output_devices()?
-        .find(|d| d.name().unwrap_or_default().contains(sink_name))
-        .context("Virtual Audio Cable, VB-CABLE or VoiceMeeter not found. Please install or reinstall one")?;
-
-    if let Some(source_name) = VIRTUAL_MICROPHONE_PAIRS.get(sink_name) {
-        Ok((
-            sink,
-            host.input_devices()?
-                .find(|d| d.name().unwrap_or_default().contains(source_name))
-                .context("Matching output microphone not found. Did you rename it?")?,
-        ))
-    } else {
-        unreachable!("Invalid argument")
-    }
-}
-
 #[allow(dead_code)]
 pub struct AudioDevice {
     inner: Device,
-    is_output: bool,
 }
```

Comment:
If this has only the `inner` field, then it could be replaced by `cpal::Device` itself